roles/openshift_logging_elasticsearch/templates/pvc.j2

```diff
@@ -25,3 +25,6 @@ spec:
   resources:
     requests:
       storage: {{size}}
+{% if storage_class_name is defined %}
+  storageClassName: ""
```

Comment:
@gnufied, since we are setting it to `""` only when static it doesn't really matter.  I went ahead and updated that piece of code in case we want it.